src/sentry/static/sentry/app/views/settings/components/emptyMessage.tsx

```diff
@@ -49,7 +50,25 @@ const Action = styled('div')`
   ${MarginStyles};
 `;
 
-const EmptyMessage = ({title, description, icon, children, action, size, className}) => (
+type Props = {
+  title?: React.ReactNode;
+  description?: React.ReactNode;
+  icon?: string;
+  action?: React.ReactElement;
+  size?: 'large' | 'medium';
+};
+
+type EmptyMessageProps = Omit<React.HTMLProps<HTMLDivElement>, keyof Props> & Props;
```

Comment:
This shouldn't have the HTML props here since we don't actually pass those into the `Wrapper`

Reply:
Though actually why don't we just spread props into the `Wrapper` so we can use additional HTML props :)

Reply:
Then we can get rid of `className` too.